frontend/src/lib/components/Projects/ProjectFilter.svelte

```diff
@@ -169,5 +176,16 @@
         </label>
       </div>
     {/if}
+    {#if filterEnabled('emptyProjects')}
+      <div class="form-control">
+        <label class="cursor-pointer label gap-4">
+          <span class="label-text inline-flex items-center gap-2">
+            {$t('project.filter.show_empty')}
+            <!-- Icon -->
```

Comment:
A good icon to use would be i-mdi-file-hidden, with no color specified.

Reply:
I like the icon suggestion 🚀 
I'd be inclined to even go for the i-mdi-folder-hidden.

Reply:
No worries!